### DISCUSSION TITLE
Overflex

### DISCUSSION THREAD

**Discussion by Ifeanyi2003** [blockscout/blockscout#11483] [Support]
Please i withdrew my USDT ERC20 to bingX but i am yet to receive it

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.

**Comment by Ifeanyi2003** [comment]
Good morning Please I swapped my Over to USDT on the app then I withdrew to
my bingx exchange but I am yet to receive it
